packages/gitbeaker-core/rollup.config.js

```diff
@@ -14,5 +15,10 @@ export default {
       format: 'es',
     },
   ],
-  plugins: commonPlugins,
+  plugins: [
+    replace({
+      __apiMap__: JSON.stringify(require('./dist/map.json'))
```

Comment:
Couldn't we also parse the JSON right here to avoid doing it in runtime? It would be faster afaik

Reply:
It does a string replace, not a variable replace. So if i tried to return a object, it would be an object wrapped in quotes. Unless im missing something